src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -409,7 +377,7 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       }
   }
 
-  val attributeList = AttributeValueList(Seq(AttributeString("a"), AttributeString("b"), AttributeBoolean(true)))
+  val attributeList = AttributeValueList(Seq(AttributeString("a"), AttributeString("b"), AttributeString("c")))
```

Comment:
why was this changed?  We should be sure mixed-type lists like this work, if we support them